### DISCUSSION TITLE
question imputation joint model

### DISCUSSION THREAD

**Discussion by amirabadiza921** [amices/mice#539] [Impute, analyse and pool]
Is it possible to use  joint models with mice package?

**Comment by hanneoberman** [comment]
Yes, see e.g. the [`jomoImpute` method](https://github.com/amices/mice/blob/master/R/mice.impute.jomoImpute.R) or [this overview](https://amices.org/mice/reference/index.html) of imputation methods implemented in `mice`.

**Comment by amirabadiza921** [comment]
Is it possible to combine the mixed effects and survival model with this
package?

‫‪Hanne Oberman‬‏ ***@***.***‬‏> در تاریخ چهارشنبه ۲۲ فوریه
۲۰۲۳ ساعت ۱۲:۴۵ نوشت:‬

**Reply by stefvanbuuren** [reply]
For mixed/multilevel models, see <https://stefvanbuuren.name/fimd/sec-threeformulations.html>. 

If your new to `mice`, then I would suggest starting in the FCS framework, which is easier and more flexible than joint models.

**Comment by hanneoberman** [comment]
I'm converting this to the Discussion page as this is not a software issue.
